.github/workflows/push-content.yml

```diff
@@ -0,0 +1,22 @@
+name: Push content
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - '1.0'
+    paths:
+      - 'modules/**'
+      - 'antora.yml'
+      - '.github/workflows/push-content.yml'
```

Comment:
Please don't do this, use the final version of the version or remove the file until the doc-site infrastructure is ready

Reply:
Ok, I'll remove it